trade-finance-manager-ui/server/controllers/utilisation-reports/add-to-an-existing-payment/get-back-link-href.ts

```diff
@@ -0,0 +1,29 @@
+// "Recommended Security and Performance Max: 2048 CHARACTERS" - https://stackoverflow.com/a/48230425
+const MAX_URL_CHARACTERS = 2048;
+const BASE_URL_CHARACTER_ALLOWANCE = 100;
+
+export const getBackLinkUrl = (reportId: string, feeRecordIds: number[]): string => {
```

Comment:
Given this is exported I would prefer if we try and be explicit in the name about which page it is for or where the link is going in case anyone ever tries to reuse it without noticing it's specific folder location